Yulia_Silko.playground/Contents.swift

```diff
@@ -1,3 +1,66 @@
 import UIKit
 
 var str = "Hello, playground"
+
+enum nitro {
+    case nitroOn
+    case nitroOff
+}
+
+enum luggage {
+    case trunkIsFull
+    case TrunkIsEmpty
+}
+
+class Car {
+    var brand: String
+    var release: Int
+    init (brand: String,
+         release: Int) {
+        self.brand = brand
+        self.release = release
+    }
+}
+
+class trunkCar: Car {
```

Comment:
```suggestion
final class TrunkCar: Car {
```

почему объект с маленькой буквы?) ужас